.github/workflows/server-test-template.yml

```diff
@@ -74,15 +74,18 @@ jobs:
             server/report.xml
       - name: Publish test report
         id: report
-        uses: mikepenz/action-junit-report@ec3a351c13e080dc4fa94c49ab7ad5bf778a9668 # v3.7.7 + count retries + check urls from https://github.com/lieut-data/action-junit-report
+        uses: mikepenz/action-junit-report@a427a90771729d8f85b6ab0cdaa1a5929cab985d # v5.0.0
         if: success() || failure() # always run even if the previous step fails
         with:
           report_paths: server/report.xml
           check_name: ${{ inputs.name }} (Results)
           job_name: ${{ inputs.name }}
           require_tests: true
+          check_retries: true
+          flaky_summary: true
+          include_passed: true
       - name: Report retried tests via webhook (master || release-*)
-        if: ${{ steps.report.outputs.retried > 0 && (github.ref_name == 'master' || startsWith(github.ref_name, 'release-')) }}
+        if: ${{ steps.report.outputs.flaky_summary != '<table><tr><th>Test</th><th>Retries</th></tr></table>' && (github.ref_name == 'master' || startsWith(github.ref_name, 'release-')) }}
```

Comment:
Oh wow, why is the `retried` counter removed?

Reply:
It's no longer present, but I'll try to submit a PR upstream so this quick hack can be reverted.

I know @lieut-data had the changes at https://github.com/lieut-data/action-junit-report -- did those eventually make it into the upstream repo?

Reply:
@saturninoabril, I regret I haven't submitted those changes upstream yet. I can try updating https://github.com/lieut-data/action-junit-report with the latest code as a stop-gap for this effort, and then submit upstream in the hopes of breaking this dependency juggling altogether?